numalogic/registry/redis_registry.py

```diff
@@ -118,7 +162,7 @@ def __save_artifact(
             mapping={
                 "artifact": serialized_artifact,
                 "version": str(version),
-                "timestamp": int(time.time()),
+                "timestamp": float(time.time()),
```

Comment:
Any reason why we are changing to float? `time.time()` will anyway return a floating value

Reply:
That is redundant!
Let me remove that